db.tf

```diff
@@ -56,7 +60,7 @@ resource "aws_rds_cluster" "apiary_cluster" {
   db_subnet_group_name         = "${aws_db_subnet_group.apiarydbsg.name}"
   vpc_security_group_ids       = ["${aws_security_group.db_sg.id}"]
   tags                         = "${var.apiary_tags}"
-  final_snapshot_identifier    = "${local.instance_alias}-cluster-final"
+  final_snapshot_identifier    = "${local.instance_alias}-cluster-final-${random_id.snapshot_id.hex}"
```

Comment:
worth adding `copy_tags_to_snapshot = true`. This will copy tags to snapshots and makes easy to identify.

Reply:
@pradeepbhadani I'll create a new PR for copy_tags_to_snapshot = true.